TMDbLib/Objects/TvShows/TvEpisodeBase.cs

```diff
@@ -28,5 +28,8 @@ public class TvEpisodeBase : TvEpisodeInfo
 
         [JsonProperty("vote_count")]
         public int VoteCount { get; set; }
+
+        [JsonProperty("runtime")]
+        public int Runtime { get; set; }
```

Comment:
Runtime can be null if the episode is unaired
```suggestion
        public int? Runtime { get; set; }
```